src/figuresClasses.ts

```diff
@@ -1,11 +1,103 @@
-export interface Figure {}
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
```

Comment:
[CODE STYLE] - Enum constant names usually start with a capital letter. Please adjust the names in the Shape enum to start with a capital letter.